test/frontend.js

```diff
@@ -88,11 +88,69 @@ describe('Frontend', function () {
     })
   })
 
+  describe('Sign up Test', function () {
+    let randInt = (min, max) => Math.floor(Math.random() * (max - min)) + min
+    let username = `testuser${randInt(0, 10000000)}${randInt(10000000, 20000000)}`
+    it('Should register User', async function () {
+      this.timeout(10000)
```

Comment:
Tests should never need such a large timeout, I would set this to 4000, max. If a test takes longer it's usually an indication of a really special circumstance or something wrong.